### DISCUSSION TITLE
你好,请问一下这个中文版要怎么安装啊?

### DISCUSSION THREAD

**Discussion by taurusduan** [inwinter04/win11React_CN#26] [Q&A]
不知道有没有简单的安装教程啊.

**Comment by inwinter04** [comment]
这是一个react项目，我并不能理解你说的安装是什么意思。
如果你希望将这个项目部署到你的云服务器上，可以参考一般react的部署方法https://blog.csdn.net/qq_42992704/article/details/121390866
当然，我作为推荐的是使用Vercel直接进行部署(事实上，我的演示站正是用这种方式部署的）。在本项目Readme页面中点击Vercel的部署图标，紧接着登录或注册你的vercel账号，绑定你的Github，剩下的事情就交给Vercel自动完成🥰

如果还有问题的话可以在此帖进行回复😎

**Reply by taurusduan** [reply]
抱歉,我是一个小白,我想部署在服务器上,然后打包成docker镜像,因为我看到只有英文的镜像.

但我不是太懂的,我用ChatGPT辅助确实完成了部署,但是问题很多.主要问题应该是ChatGPT也不知道哪个目录是项目,所以默认更目录打包...

我自己是想用这个桌面创建链接来管理本地的docker项目.

**Reply by inwinter04** [reply]
> 抱歉,我是一个小白,我想部署在服务器上,然后打包成docker镜像,因为我看到只有英文的镜像.
> 
> 但我不是太懂的,我用ChatGPT辅助确实完成了部署,但是问题很多.主要问题应该是ChatGPT也不知道哪个目录是项目,所以默认更目录打包...
> 
> 我自己是想用这个桌面创建链接来管理本地的docker项目.

我并没有使用过Docker，但是我觉得如果直接拉取本仓库的代码就会是中文的。Readme文件中的Docker部署方式为原项目的，所以没有汉化。由于我不会Docker，所以那个地方就一直没更改。
